docs/usage/advanced_configuration.rst

```diff
@@ -0,0 +1,27 @@
+You can override default values of service deployment via ansible group_vars:
+
+Ansible let you override configuration via mutliple ways
+
+Via group_vars directly:
+
+:file:`{{ inventory_dir }}/group_vars/kube-master/extra_config.yml`:
+
+    nginx_ingress_external_values:
+    - rbac:
+        create: False
+
+
+You can also tell ansible to load static file:
+
+:file:`/config/a.yaml`:
+
+    rbac:
+      create: False
+
+:file:`{{ inventory_dir }}/group_vars/kube-master/extra_config.yml`:
+
+    nginx_ingress_external_values:
+    - "{{ lookup('file', '/config/a.yml')|from_yaml }}"
```

Comment:
If https://github.com/scality/metalk8s/pull/291/files#r212996776, this will need to be changed as well of course.